target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -96,6 +96,9 @@ ramips_setup_interfaces()
 	tplink,tl-wpa8631p-v3)
 		ucidef_set_interface_lan "lan1 lan2 lan3 plc0"
 		;;
+	tplink,mr600-v2-eu)
+		ucidef_set_interfaces_lan_wan "lan1 lan2 lan3" "wan"
```

Comment:
There is no port named wan in dts?

Reply:
Hey the `wan` port is covered [here](https://github.com/openwrt/openwrt/pull/10887/files#diff-6b77468626909ff021fe50c7b8fbc364245df0bc015e0fa50c7a401d7db198c7R165). I've tried to replicate what this [PR](https://github.com/openwrt/openwrt/pull/10238) did, in order to mux all ports under `gmac1` for more bandwidth.